### DISCUSSION TITLE
2022-09-02 OMI Champions Meeting

### DISCUSSION THREAD

**Discussion by humbletim** [omigroup/omigroup#288] [Weekly Meetings]
(humbletim suggestions please chime in to suggest other topics)

* top-level org review?
  - subgroup existence check-ins (and noting if/where docs need to be updated).
  - OMI readme.md.
  - OMI website.
  - OMI legal status check-in
    - what kind of org are we?
    - anything on the radar next?
  - OMI "champion" check-in
    - what is a champion?
    - what is the currently expected participation level?
    - who are the champions?
    - does that list reflect everyone who is significantly helping champion?

* mini retrospective?

* recent notable mentions?
  - redeagle has asked for help welcoming new users to the discord.
  - ifire/jin have asked about running open source project work through OMI's opencollective.
  - mrmetaverse/ifire have been testing a refined "daily standup" concept.
    - thoughts? what would it take to formally adopt?

* champion onboarding?
  - open questions?
  - system access / known silos
    - auth roles? (email, 1password, wordpress, discord, streaming services, etc.)
    - managing and sharing passwords?
  - ...

**Comment by futuristudios** [comment]
A's thoughts on the conversation:
- defining champions, stewards? advocates? evangelist? (define by list what is and what is not + who decides)
- capacity and category(ies) of champion
- championing by action first, then community 'vote' as post-recognition
- what's the ideal? autonomous alignment agility (lessen/mitigate bureaucracy inertia)
- distinction between the individual and a logo (reputation/good will, money and other in-kind conflicts)
- quarterly calls to discussed finances?
- …

**Comment by humbletim** [comment]
for reference https://github.com/omigroup/omigroup/discussions/211

```
mrmetaverse
robertlong
antpb
paulespino
musicscene (mtv969)
fire
RedEagleP1
madjin
LightLodges (evo)
Elirudite
humbletim
Knar33
```

> The leadership team will meet once a week for an hour to plan out the week's responsibilities and ensure our tasks are covered. As a member of the leadership team you should have at least one hour a week available to meet with the rest of the leadership team as well as some time to accomplish the various tasks.
> 
> Members of the leadership team should have good standing with the W3C Community Guidelines and should be role models for the rest of the community. They will be responsible for upholding these guidelines and fostering the growth of a respectful and inclusive community.
> 
> Leadership team members will help steer the community via the responsibilities listed above. However, all decisions within the community will still be made via a vote by consensus. Leaders will not have any more weight than any other member in the voting process. The leadership team is responsible for acting on behalf of the community and should take in and address feedback regularly.

**Comment by humbletim** [comment]
re: defining champions

jin/discord: 
- ![image](https://user-images.githubusercontent.com/94753/188224627-731e7521-92d1-42c3-b708-dfe71eb00143.png)
- https://metaverse-standards.org/members/ (reference point Member vs. Principle)
- ...

**Comment by madjin** [comment]
## OMI "champion" check-in
- https://github.com/omigroup/omigroup/discussions/211 OMI Election

Few folks that have been inactive since the election, can reach out to them to check-in:
- https://github.com/omigroup/omigroup/discussions/211#discussioncomment-2560998
- https://github.com/omigroup/omigroup/discussions/211#discussioncomment-2602375
- https://github.com/omigroup/omigroup/discussions/211#discussioncomment-2560945 

**What is a OMI champion?**
[OMI Election thread](https://github.com/omigroup/omigroup/discussions/211) has lots of good snippets but it seems that consensus is much needs to be revised.

iFire made a good suggestion to look at MSF for inspiration: https://metaverse-standards.org/members/
There are two types of members
- **Principle members**
  - Principal organizations are those that also wish to assist in Forum oversight and may wish to fund Forum projects. Logos updated once a day
- **Participant**
  - Join as a Participant organizations to engage in forum discussions and projects. Logos updated once a day.

**Comment by humbletim** [comment]
shadow via discord: 

> Champion - individuals in the OMIgroup who have been elected to take on leadership roles

> Allies - Groups who align with OMIgroup principles

> Sponsors - Those who contribute non-financial resources to OMIgroup

> Donors - Financial supporters of OMIgroup

**Reply by Elirudite** [reply]
Updated based on our conversation on it:

Champion - individuals in the OMIgroup who have been elected to take on leadership roles

Allies/Fellows/Friends - Groups who align with OMIgroup principles

Sponsors - Contributors to OMIgroup who expect something in return (such as recognition)
Event Sponsors only relate to specific events.

Donors - Financial supporters of OMIgroup who do not expect anything in return

**Reply by Elirudite** [reply]
Definition updated based on input from ▲.#1802

Champion - individuals in the OMIgroup who have been elected into a leadership role after displaying ability to execute on tasks and initiatives aligned with the group.

**Comment by madjin** [comment]
cont...
We want to avoid the word 'partnerships', but also would like to have representation of friends / allies on the website. We would like to have something at the bottom of the OMI website where we could have graphics of individuals or orgs that align with OMI, but first need to agree on the process is for including such.
![image](https://user-images.githubusercontent.com/32600939/188225583-25af01ad-cc33-4b52-b3f5-126b88e095ed.png)

Maybe opportunity for giving on the website with representation of such on the site like Blender?
https://fund.blender.org/: https://download.blender.org/institute/BF_Development_Fund_Membership.pdf

Donors would be separate from Sponsors, Donors are individuals or orgs who expect nothing in return

**Comment by humbletim** [comment]
evo via discord:

(rough capture re: OMI expenses)

* under $50 flexible
* over $50 requires discussion 

... revisit fiscal subgroup? monthly OMI financing checkins?

**Reply by humbletim** [reply]
- discussing / reimbursing current out-of-pocket expenses (website, etc.)

**Comment by humbletim** [comment]
reference point via jin: https://download.blender.org/institute/BF_Development_Fund_Membership.pdf
